kclvm/config/src/modfile.rs

```diff
@@ -2,13 +2,31 @@
 
 use kclvm_utils::path::PathPrefix;
 use serde::Deserialize;
-use std::io::Read;
+use std::{env, io::Read};
 use toml;
 
 pub const KCL_MOD_FILE: &str = "kcl.mod";
 pub const KCL_FILE_SUFFIX: &str = ".k";
 pub const KCL_FILE_EXTENSION: &str = "k";
 pub const KCL_MOD_PATH_ENV: &str = "${KCL_MOD}";
+pub const VENDOR_HOME: &str = "VENDOR_HOME";
```

Comment:
Use `KCLVM_VENDOR_HOME` to prevent conflicts like `RUSTC_XX`?

Reply:
fixed.